packages/overlays/src/LocalOverlayController.js

```diff
@@ -1,36 +1,77 @@
 import { render, html } from '@lion/core';
-import { managePosition } from './utils/manage-position.js';
 import { containFocus } from './utils/contain-focus.js';
 import { keyCodes } from './utils/key-codes.js';
 
 export class LocalOverlayController {
   constructor(params = {}) {
-    const finalParams = {
+    const defaultOptions = {
       placement: 'top',
-      position: 'absolute',
-      ...params,
+      positionFixed: false,
+      modifiers: {
+        keepTogether: {
+          enabled: false,
+        },
+        preventOverflow: {
+          enabled: true,
+          boundariesElement: 'viewport',
+          padding: 16, // viewport-margin for shifting/sliding
+        },
+        flip: {
+          boundariesElement: 'viewport',
+          padding: 16, // viewport-margin for flipping
+        },
+        offset: {
+          enabled: true,
+          offset: `0, 8px`, // horizontal and vertical margin (distance between popper and referenceElement)
+        },
+        arrow: {
+          enabled: false,
+        },
+      },
     };
-    this.hidesOnEsc = finalParams.hidesOnEsc;
-    this.hidesOnOutsideClick = finalParams.hidesOnOutsideClick;
-    this.trapsKeyboardFocus = finalParams.trapsKeyboardFocus;
-    this.placement = finalParams.placement;
-    this.position = finalParams.position;
+
+    const finalPlacement = {
+      ...defaultOptions,
+      ...params.placementConfig,
+    };
+
+    let finalModifiers = defaultOptions.modifiers;
+    if (params.placementConfig) {
+      finalModifiers = {
+        ...finalModifiers,
+        ...params.placementConfig.modifiers,
+      };
+    }
+
+    this.placementConfig = {
+      placement: finalPlacement.placement,
+      positionFixed: finalPlacement.positionFixed,
+      modifiers: finalModifiers,
+    };
```

Comment:
This is quite hard to read for me :) Can we replace all of the above with this?

```js
    this.placementConfig = {
      placement: 'top',
      positionFixed: false,
      ...(params.placementConfig  || {}),
      modifiers: {
        keepTogether: {
          enabled: false,
        },
        preventOverflow: {
          enabled: true,
          boundariesElement: 'viewport',
          padding: 16, // viewport-margin for shifting/sliding
        },
        flip: {
          boundariesElement: 'viewport',
          padding: 16, // viewport-margin for flipping
        },
        offset: {
          enabled: true,
          offset: `0, 8px`, // horizontal and vertical margin (distance between popper and referenceElement)
        },
        arrow: {
          enabled: false,
        },
        ...(params.placementConfig && params.placementConfig.modifiers  || {}),
      },
    };
```

Reply:
Whoa this is some Javascript spread syntax magic I was not aware of, works fine and is indeed much more readable